common/src/main/java/com/wynntils/handlers/item/ItemHandler.java

```diff
@@ -172,6 +179,34 @@ private boolean isWildcardItem(ItemStack itemStack) {
         return WILDCARD_ITEMS.contains(itemStack.getItem());
     }
 
+    /**
+     * This checks if the lore of the second item contains the entirety of the first item's lore, or vice versa.
+     * It might have additional lines added, but these are not checked.
+     */
+    private boolean isLoreSoftMatching(ItemStack firstItem, ItemStack secondItem) {
+        List<StyledText> firstLines = LoreUtils.getLore(firstItem);
+        List<StyledText> secondLines = LoreUtils.getLore(secondItem);
+        int firstLinesLen = firstLines.size();
+        int secondLinesLen = secondLines.size();
+
+        // Only allow a maximum number of additional lines in the longer tooltip
+        if (Math.abs(firstLinesLen - secondLinesLen) > 3) return false;
+
+        int linesToCheck = Math.min(firstLinesLen, secondLinesLen);
+        // Prevent soft matching on tooltips that are very small
+        if (linesToCheck < 3 && firstLinesLen != secondLinesLen) return false;
```

Comment:
Why are these needed? Are there specific items that meet these criteria?

Reply:
This is not new code; it is moved from LoreUtils. 

I am not sure if they are needed; I added this long time ago to not give false positives for items with very short lores. But it is possible that at this point, the "soft" matching could be replaced with a hard match, since that is basically how we treat the result of this method anyway. I'd rather do that separately, though, since it is always soooo scary to change anything here. Things can break in very unexpected ways.